src/settings/settings.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/sequelize';
+import { CreateSettingDto } from './dto/create-setting.dto';
+import { UpdateSettingDto } from './dto/update-setting.dto';
+import Setting from '../models/setting.model';
+
+@Injectable()
+export class SettingsService {
+  constructor(
+    @InjectModel(Setting)
+    private settingModel: typeof Setting,
+  ) {}
+
+  async create({ name, account_id, data_type, value }: CreateSettingDto) {
+    if (typeof value === 'object') value = JSON.stringify(value);
```

Comment:
Don't we have the same problem for booleans and numbers? Or are they getting implicitly converted to strings before being sent to postgres in the query?

Reply:
Implicitly converted through locally testing